docs/api/LightroomPartnerAPIsSpec.json

```diff
@@ -1110,6 +1110,116 @@
             "$ref":"#/components/responses/ServiceUnavailable"
           }
         }
+      },
+      "post": {
+        "tags": [
+          "albums"
+        ],
+        "summary": "Update album",
+        "description": "Update an existing album.",
+        "operationId": "updateAlbum",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "X-API-Key",
+            "in": "header",
+            "description": "Client ID (API Key) which is subscribed to the Lightroom APIs through console.adobe.io",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "Authorization",
+            "in": "header",
+            "description": "Bearer [token] - User access token of an authenticated Lightroom customer",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "catalog_id",
+            "in": "path",
+            "description": "Identifier of the catalog containing the album.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "album_id",
+            "in": "path",
+            "description": "Identifier for the album.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Album metadata and information to be updated.",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "subtype": {
+                  "type": "string",
+                  "enum": [ "project", "project_set" ]
+                },
+                "serviceId": {
+                  "type": "string",
+                  "description": "The API Key (client identifier) of the service creating the album."
+                },
+                "payload": {
```

Comment:
changes in payload should not be limited. anything inside payload can be updated.